vercel/turborepo
# feat: add turbo.json schema to turbo package

### Description

Implements #4878

This PR copies the schema.json from the docs and includes it in the `turbo` package.

I went with a naive copy to keep this simple and avoid any weird issues symlinks might introduce. Based on the fact that the schema doesn't change much I also instrumented tests against drift instead of automatically keeping them in sync.

### Testing Instructions

Verify that `schema.json` gets included in the tar we send to npm
```
$ cd packages/turbo && pnpm pack --pack-destination=/tmp
$ cd /tmp
$ tar xzf turbo-1.10.0-canary.0.tgz
$ ls packages
LICENSE                 bin                     node-platform.js        schema.json
README.md               install.js              package.json
```
